spec/models/placements/mentor_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe Placements::Mentor, type: :model do
+  context "with associations" do
+    it { is_expected.to have_many(:mentor_memberships) }
+    it { is_expected.to have_many(:schools).through(:mentor_memberships) }
+  end
+
+  describe "default scope" do
+    it "is scoped to placements mentors" do
+      mentor = create(:placements_mentor)
+      create(
+        :mentor_membership,
+        :placements,
+        mentor:,
+      )
+
+      expect(described_class.all).to contain_exactly(mentor)
+    end
```

Comment:
Ditto, re: testing the scope excludes non-placements mentors

Reply:
c06a141d8659b694ad202fd79569d29eaeceaf2c